Clinical Key.js

```diff
@@ -141,116 +141,95 @@ function doWeb(doc, url){
 		search.getTranslators();
 	}
 	else if (contentType == 'journalArticle') {
-		eid = url.split('/');
-		pii = eid.pop().slice(7);
-		
-		/* can use xpath to get pii; guessing the url is faster
-		var eid = ZU.xpathText(doc, "//ul/@data-eid");
-		pii = eid.slice(7);*/
-		
-		if(!/^S(\d{15}X|\d{16})/.test(pii)){
-			throw new Error('PII incorrectly formatted');
-			return;
-		}
-		
+		try {
+			eid = url.split('/');
+			pii = eid.pop().slice(7);
+			if (!/^S(\d{15}X|\d{16})/.test(pii)){
+				throw new Error('PII from url failed. Trying Xpath');
+			}
+			} catch(e) {
+				Zotero.debug(e);
+				eid = ZU.xpathText(doc, "//ul/@data-eid");
+				pii = eid.slice(7);
+				if (!/^S(\d{15}X|\d{16})/.test(pii)){
+					throw new Error('PII from Xpath failed');
+				}
+			}
 		doSearch(pii, doc);
 	}
 	
 }
 
 //Search & Processing based on CrossRef.js translator
 function doSearch(pii, doc){
-	crossRefQuery = 'http://api.crossref.org/works?query=' + pii
+	crossRefQuery = 'http://api.crossref.org/works?query=' + pii;
+	//acceptHeader = {'Accept': 'application/vnd.crossref-api-message+json; version=1.0'}
```

Comment:
Hoping for some feedback here with managing API versions for the CrossRef REST. They have documentation [here](https://github.com/CrossRef/rest-api-doc/blob/master/rest_api.md#how-to-manage-api-versions) specifying version requests. However, I couldn't get it to work ([link to the issue](https://github.com/CrossRef/rest-api-doc/issues/181)). It seems to be server side based on my GET request (see the issue), but I could be wrong.